scripts/bootstrap-py3-venv

```diff
@@ -22,13 +24,20 @@ command -v direnv >/dev/null || {
     return 1
 }
 
-export SENTRY_PYTHON3=1
-
 gitroot="$(git rev-parse --show-toplevel)"
 cd "$gitroot"
+export venv_name="${PWD}/.venv"
+
+if [[ -f "${venv_name}/bin/activate" ]]; then
+    prompt_text="You seem to have a virtualenv already. This script is meant to start one from scratch. Continue (y/N)?"
+
+    # Temporary special case for Python 2 .venv/ -> .venv2/ transition.
+    # Just a more helpful error message.
+    if [[ "$(python -V 2>&1)" = "Python $(grep "2.7" .python-version)" ]]; then
+        prompt_text="You seem to have a virtualenv already, but it's Python 2. Remove it and replace with Python 3 (y/N)?"
```

Comment:
```
You have an existing Python 2 virtualenv. Remove it and replace with a Python 3 virtualenv? (y/N)
```

Reply:
Recommended to use the suggested changes feature, it also pretty prints a diff.